src/model/object/entity/individual/individual.h

```diff
@@ -8,7 +8,7 @@
 #include "../entity.h"
 
 class Individual : public Entity {
-private:
+protected:
```

Comment:
I think this should stay private; fred or any other subclass doesn't need to know the ID

Reply:
ahh ok, i see. But shouldn't then other member variables be private, too(such as position in Object, or energy in Entity)?

Reply:
the difference is, the ID is metadata - data about the object.
it is nothing the object should ever be able to read

Reply:
ok, so that basically means:
private members do only make sense within abstract classes in few cases.
one of these cases could be an constant ID, that is set once during construction.